gtd

```diff
@@ -119,9 +121,13 @@ vsleep() {
    while [[ "$left" -gt 0 ]]; do
       hms="$(hms $left)"
 
-      # Updates a temporary file for tmux to read from and refreshes the client
+      # Updates a temporary file for external scripts
+      if [[ -n "$DO_TMUX" || -n "$DO_TMP" ]]; then
+         (echo -ne "$hms" > /tmp/gtd)
```

Comment:
I removed the pipe, because I feel like this type of formatting should be handled by the user outside of the script. This is just a personal preference thing so just let me know how you feel about it.